src/workerd/util/sqlite-test.c++

```diff
@@ -663,6 +663,36 @@ KJ_TEST("SQLite write row counters (basic)") {
   }
 }
 
+KJ_TEST("SQLite read/write row counters (large row insert)") {
```

Comment:
It would be good to comment why we have this test (either in the commit message or with a comment here).

I believe it's to make sure that extra page reads aren't included in the row count.

Does it also make sense to have more than one row for this test?  I don't know how big rows are actually stored in SQLite.

Reply:
Big rows are stored  in what is called overflow pages - https://www.sqlite.org/fileformat2.html#ovflpgs

I don't think having multiple rows in the test is going to give us any new information.

Will add the comment